common/main/fil.xml

```diff
@@ -2331,61 +2331,61 @@ Warnings: All cp values have U+FE0F characters removed. See /annotationsDerived/
+							<dayPeriod type="evening1">ng gabi</dayPeriod>
 							<dayPeriod type="night1">ng gabi</dayPeriod>
 						</dayPeriodWidth>
 					</dayPeriodContext>
 					<dayPeriodContext type="stand-alone">
 						<dayPeriodWidth type="abbreviated">
-							<dayPeriod type="midnight">↑↑↑</dayPeriod>
-							<dayPeriod type="am">↑↑↑</dayPeriod>
-							<dayPeriod type="noon">↑↑↑</dayPeriod>
-							<dayPeriod type="pm">↑↑↑</dayPeriod>
+							<dayPeriod type="midnight">hatinggabi</dayPeriod>
+							<dayPeriod type="am">AM</dayPeriod>
+							<dayPeriod type="noon">tanghaling-tapat</dayPeriod>
+							<dayPeriod type="pm">PM</dayPeriod>
 							<dayPeriod type="morning1">umaga</dayPeriod>
-							<dayPeriod type="morning2">↑↑↑</dayPeriod>
+							<dayPeriod type="morning2">madaling-araw</dayPeriod>
 							<dayPeriod type="afternoon1">hapon</dayPeriod>
 							<dayPeriod type="evening1">gabi</dayPeriod>
-							<dayPeriod type="night1">↑↑↑</dayPeriod>
+							<dayPeriod type="night1">gabi</dayPeriod>
 						</dayPeriodWidth>
 						<dayPeriodWidth type="narrow">
-							<dayPeriod type="midnight">↑↑↑</dayPeriod>
-							<dayPeriod type="am">↑↑↑</dayPeriod>
-							<dayPeriod type="noon">↑↑↑</dayPeriod>
-							<dayPeriod type="pm">↑↑↑</dayPeriod>
-							<dayPeriod type="morning1">↑↑↑</dayPeriod>
-							<dayPeriod type="morning2">↑↑↑</dayPeriod>
-							<dayPeriod type="afternoon1">↑↑↑</dayPeriod>
-							<dayPeriod type="evening1">↑↑↑</dayPeriod>
-							<dayPeriod type="night1">↑↑↑</dayPeriod>
+							<dayPeriod type="midnight">hatinggabi</dayPeriod>
+							<dayPeriod type="am">AM</dayPeriod>
+							<dayPeriod type="noon">tanghaling-tapat</dayPeriod>
+							<dayPeriod type="pm">PM</dayPeriod>
+							<dayPeriod type="morning1">umaga</dayPeriod>
+							<dayPeriod type="morning2">madaling-araw</dayPeriod>
+							<dayPeriod type="afternoon1">hapon</dayPeriod>
+							<dayPeriod type="evening1">gabi</dayPeriod>
+							<dayPeriod type="night1">gabi</dayPeriod>
 						</dayPeriodWidth>
 						<dayPeriodWidth type="wide">
-							<dayPeriod type="midnight">↑↑↑</dayPeriod>
-							<dayPeriod type="am">↑↑↑</dayPeriod>
-							<dayPeriod type="noon">↑↑↑</dayPeriod>
-							<dayPeriod type="pm">↑↑↑</dayPeriod>
-							<dayPeriod type="morning1">↑↑↑</dayPeriod>
-							<dayPeriod type="morning2">↑↑↑</dayPeriod>
-							<dayPeriod type="afternoon1">↑↑↑</dayPeriod>
-							<dayPeriod type="evening1">↑↑↑</dayPeriod>
-							<dayPeriod type="night1">↑↑↑</dayPeriod>
+							<dayPeriod type="midnight">hatinggabi</dayPeriod>
+							<dayPeriod type="am">AM</dayPeriod>
+							<dayPeriod type="noon">tanghaling-tapat</dayPeriod>
+							<dayPeriod type="pm">PM</dayPeriod>
+							<dayPeriod type="morning1">umaga</dayPeriod>
+							<dayPeriod type="morning2">madaling-araw</dayPeriod>
+							<dayPeriod type="afternoon1">hapon</dayPeriod>
+							<dayPeriod type="evening1">gabi</dayPeriod>
+							<dayPeriod type="night1">gabi</dayPeriod>
```

Comment:
There are 4 errors in fil.xml, all for the value "gabi": lines 2387, 2388, 2365, 2366.

/Users/tbishop/Documents/WenlinDocs/Organizations/Unicode/CLDR_job/cldr/common/main/fil.xml:2387:51: error
fil [Filipino]	error	▸Date_&_Time|Gregorian|Day_Periods_-_wide_-_Standalone|evening1◂	〈evening〉	【】	〈gabi〉	«=»	【】	⁅date symbol collision⁆	❮Error: The date value “gabi” is the same as what is used for a different item: [Day Periods - abbreviated - Standalone: night1, Day Periods - wide - Standalone: night1]. Sample problem: 17❯	https://st.unicode.org/cldr-apps/v#/fil//5898cea3b011ee03
/Users/tbishop/Documents/WenlinDocs/Organizations/Unicode/CLDR_job/cldr/common/main/fil.xml:2388:49: error
fil [Filipino]	error	▸Date_&_Time|Gregorian|Day_Periods_-_wide_-_Standalone|night1◂	〈night〉	【】	〈gabi〉	«=»	【】	⁅date symbol collision⁆	❮Error: The date value “gabi” is the same as what is used for a different item: [Day Periods - abbreviated - Standalone: evening1, Day Periods - wide - Standalone: evening1]. Sample problem: 17❯	https://st.unicode.org/cldr-apps/v#/fil//5399bec30a01dabd
/Users/tbishop/Documents/WenlinDocs/Organizations/Unicode/CLDR_job/cldr/common/main/fil.xml:2365:51: error
fil [Filipino]	error	▸Date_&_Time|Gregorian|Day_Periods_-_abbreviated_-_Standalone|evening1◂	〈evening〉	【】	〈gabi〉	«=»	【】	⁅date symbol collision⁆	❮Error: The date value “gabi” is the same as what is used for a different item: [Day Periods - abbreviated - Standalone: night1, Day Periods - wide - Standalone: night1]. Sample problem: 17❯	https://st.unicode.org/cldr-apps/v#/fil//5467a53972631647
/Users/tbishop/Documents/WenlinDocs/Organizations/Unicode/CLDR_job/cldr/common/main/fil.xml:2366:49: error
fil [Filipino]	error	▸Date_&_Time|Gregorian|Day_Periods_-_abbreviated_-_Standalone|night1◂	〈night〉	【】	〈gabi〉	«=»	【】	⁅date symbol collision⁆	❮Error: The date value “gabi” is the same as what is used for a different item: [Day Periods - abbreviated - Standalone: evening1, Day Periods - wide - Standalone: evening1]. Sample problem: 17https://st.unicode.org/cldr-apps/v#/fil//22437ed5ce3f9707

43 errors in bs-Cyrl plus 4 errors in fil = total 47 errors for this PR

Reply:
The Gabi error is actually already known in this ticket: CLDR-17035

Reply:
4 paths with test failures when values are "hardened":

https://st.unicode.org/cldr-apps/v#/fil/Gregorian/5399bec30a01dabd
https://st.unicode.org/cldr-apps/v#/fil/Gregorian/22437ed5ce3f9707
https://st.unicode.org/cldr-apps/v#/fil/Gregorian/5898cea3b011ee03
https://st.unicode.org/cldr-apps/v#/fil/Gregorian/5467a53972631647

----

https://st.unicode.org/cldr-apps/v#/fil/Gregorian/5898cea3b011ee03

//ldml/dates/calendars/calendar[@type="gregorian"]/dayPeriods/dayPeriodContext[@type="stand-alone"]/dayPeriodWidth[@type="wide"]/dayPeriod[@type="evening1"]

inherits from

https://st.unicode.org/cldr-apps/v#/fil/Gregorian/5467a53972631647

//ldml/dates/calendars/calendar[@type="gregorian"]/dayPeriods/dayPeriodContext[@type="stand-alone"]/dayPeriodWidth[@type="abbreviated"]/dayPeriod[@type="evening1"]

--------

https://st.unicode.org/cldr-apps/v#/fil/Gregorian/22437ed5ce3f9707

//ldml/dates/calendars/calendar[@type="gregorian"]/dayPeriods/dayPeriodContext[@type="stand-alone"]/dayPeriodWidth[@type="abbreviated"]/dayPeriod[@type="night1"]

inherits from 

https://st.unicode.org/cldr-apps/v#/fil/Gregorian/50fdd2c1fa3fe288

//ldml/dates/calendars/calendar[@type="gregorian"]/dayPeriods/dayPeriodContext[@type="format"]/dayPeriodWidth[@type="abbreviated"]/dayPeriod[@type="night1"]

---------

https://st.unicode.org/cldr-apps/v#/fil/Gregorian/5399bec30a01dabd

//ldml/dates/calendars/calendar[@type="gregorian"]/dayPeriods/dayPeriodContext[@type="stand-alone"]/dayPeriodWidth[@type="wide"]/dayPeriod[@type="night1"]

inherits from 

https://st.unicode.org/cldr-apps/v#/fil/Gregorian/50fdd2c1fa3fe288

//ldml/dates/calendars/calendar[@type="gregorian"]/dayPeriods/dayPeriodContext[@type="format"]/dayPeriodWidth[@type="abbreviated"]/dayPeriod[@type="night1"]

Reply:
The only reason I'm going back in time